script/testing/oltpbench/configs/nightly.json

```diff
@@ -1,5 +1,9 @@
 {
     "type": "oltpbenchmark",
+    "server_args":{
+        "wal": false,
+        "log_file_path": "/mnt/ramdisk/wal.log"
```

Comment:
I think turning `wal` to `false` means there won't be a WAL so then specifying the `log_file_path` is kind of meaningless

Reply:
I think you are right. I think there is a requirement conflict between cmu-db#1062 and cmu-db#1054